src/sdg/api/serializers/organisaties.py

```diff
@@ -110,17 +110,43 @@ class Meta:
             },
         }
 
-    def get_openingstijden(self, obj: Locatie) -> OpeningstijdenSerializer:
-        return OpeningstijdenSerializer(obj).data
-
 
 class LocatieSerializer(LocatieBaseSerializer):
     """Serializer for location details, including contact details, address and opening times."""
 
     organisatie = LokaleOverheidBaseSerializer(source="lokale_overheid")
+    openingstijden = OpeningstijdenSerializer(source="*")
 
     class Meta(LocatieBaseSerializer.Meta):
-        fields = LocatieBaseSerializer.Meta.fields + ("organisatie",)
+        fields = LocatieBaseSerializer.Meta.fields + ("organisatie", "openingstijden")
+
+    @transaction.atomic
+    def create(self, validated_data):
+        lokale_overheid = validated_data.pop("lokale_overheid")
+        initial_organisatie = lokale_overheid["organisatie"]
+
+        try:
+            validated_data["lokale_overheid"] = LokaleOverheid.objects.get(
+                organisatie__owms_pref_label=initial_organisatie["owms_pref_label"]
+            )
```

Comment:
In het ticket staat dat het ook moet werken indien `owmsIdentifier` is meegegeven.